projects/plugins/jetpack/modules/protect/blocked-login-page.php

```diff
@@ -370,7 +370,11 @@ function display_page( $title, $message, $back_button = false, $recovery_form =
 			<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 			<meta name="viewport" content="width=device-width">
 			<?php
-			if ( function_exists( 'wp_no_robots' ) ) {
+			// Remove wp_no_robots() when WordPress 5.7 is the minimum required version.
+			if ( function_exists( 'wp_robots' ) && function_exists( 'wp_robots_no_robots' ) && function_exists( 'add_filter' ) ) {
```

Comment:
Isn't `add_filter` a core function? Why do we need to check that the function exists?

Reply:
I originally didn't include it, but ended up adding that check to mirror Core's implementation:
https://core.trac.wordpress.org/changeset/49992/trunk/src/wp-includes/functions.php